openmetadata-ui/src/main/resources/ui/src/components/common/CustomPropertyTable/PropertyValue.tsx

```diff
@@ -26,14 +26,15 @@ interface Props {
   propertyType: EntityReference;
   extension: Table['extension'];
   onExtensionUpdate: (updatedExtension: Table['extension']) => Promise<void>;
+  hasEditPermissions: boolean;
 }
 
 const EditIcon = ({ onShowInput }: { onShowInput: () => void }) => (
   <span
-    className="tw-cursor-pointer tw-ml-2"
+    className="cursor-pointer m-l-xs h-auto mt-05"
     data-testid="edit-icon"
     onClick={onShowInput}>
-    <SVGIcons alt="edit" icon={Icons.EDIT} width="16px" />
+    <EditIconComponent height={14} width={14} />
```

Comment:
why we are reducing size? we follow 16x16, if it is necessary to make changes please take the approval of @devyani-kaushik 
also, we can use Button instead of span!

Reply:
@ShaileshParmar11 Button will add padding in it, so span will do our job.